ditto/readers/cyme/read.py

```diff
@@ -4539,9 +4668,9 @@ def parse_transformers(self, model):
                 XR = float(transformer_data["xr"])
                 XR0 = float(transformer_data["xr0"])
                 R1 = Z1 / math.sqrt(1 + XR * XR)
-                R0 = Z0 / math.sqrt(1 + XR * XR)
-                X1 = Z1 / math.sqrt(1 + XR0 * XR0)
-                X0 = Z0 / math.sqrt(1 + XR0 * XR0)
+                R0 = Z0 / math.sqrt(1 + XR0 * XR0)
+                X1 = Z1 / math.sqrt(1 + 1 / (XR * XR))
+                X0 = Z0 / math.sqrt(1 + 1 / (XR0 * XR0))
```

Comment:
Rename `XR0` -> `X0R0`, and `XR` -> `X1R1`

Reply:
These names come from the CYME names. We're using them directly